providers/mysqlxorm/sqlite3/interface.go

```diff
@@ -74,7 +92,29 @@ func NewSqlite3(dbSourceName string, opts ...OptionFunc) (*Sqlite3, error) {
 
 	engine.SetMapper(names.GonicMapper{})
 
-	sqlite3Engine := &Sqlite3{db: engine}
+	sqlite3Engine := &Sqlite3{
+		db: engine,
+	}
 
 	return sqlite3Engine, nil
 }
+
+func (s *Sqlite3) Close() error {
+	if atomic.CompareAndSwapInt32(&s.closeState, 0, 1) {
+		err := s.DB().Close()
+		if err != nil {
+			return err
+		}
+
+		err = os.Remove(s.DataSourceName())
+		return err
+	}
+	return nil
+}
+
+func randomName(path string) string {
+	dir, file := filepath.Split(path)
```

Comment:
use uuid directly or obey the rule of `os.CreateTemp`.

Reply:
Done.